src/deepsparse/v2/operators/engine_operator.py

```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
 
```

Comment:
General comment: for easier reviewing, a quicker feedback loop, and higher velocity, consider breaking up PRs like this into smaller logical units. I know that it is often not possible. But doing so, would allow you to sneak in at least few additional tests, and would spare us going through a PR with 16 edited scripts :)

Reply:
yah sorry, the refactor wasn't expected.